modules/tensor_mechanics/src/actions/InclinedNoDisplacementBCAction.C

```diff
@@ -0,0 +1,100 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "InclinedNoDisplacementBCAction.h"
+#include "Factory.h"
+#include "FEProblem.h"
+#include "Conversion.h"
+
+registerMooseAction("TensorMechanicsApp", InclinedNoDisplacementBCAction, "add_bc");
+
+template <>
+InputParameters
+validParams<InclinedNoDisplacementBCAction>()
+{
+  InputParameters params = validParams<Action>();
+  params.addClassDescription("Set up inclined no displacement boundary conditions");
+
+  params.addRequiredParam<std::vector<BoundaryName>>(
+      "boundary", "The list of boundary IDs from the mesh where the pressure will be applied");
+
+  params.addParam<VariableName>("disp_x", "The x displacement");
+  params.addParam<VariableName>("disp_y", "The y displacement");
+  params.addParam<VariableName>("disp_z", "The z displacement");
+
+  params.addParam<std::vector<VariableName>>(
+      "displacements",
+      "The displacements appropriate for the simulation geometry and coordinate system");
+
+  params.addParam<std::vector<AuxVariableName>>("save_in_disp_x",
```

Comment:
Same with this.  Handle the save_in variables in the same way that it is done in TensorMechanicsAction.C (single input parameter `save_in` takes a vector of variables).  It looks like the Pressure action (which I assume you based this off of) is not set up that way, but it should be.

Reply:
done!